apps/web/pages/api/v1/environments/[environmentId]/product/index.ts

```diff
@@ -114,6 +115,39 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse)
     const firstEnvironment = newProduct.environments[0];
     res.json(firstEnvironment);
   }
+    
+  // DELETE
+  else if (req.method === "DELETE") {
+    console.log({currentUser})
+    const membership = await prisma.membership.findUnique({
+      where: {
+        userId_teamId: {
+          userId: currentUser.id,
+          teamId: currentUser.teamId,
+        },
+      },
+    });
+    if (membership?.role !== "admin") {
```

Comment:
We also have the role `owner` for a team which is the default for the person created the team. So the user should be of role `admin` or `owner` to be able to delete a product.

Reply:
Yes makes sense. This is done @mattinannt.
Thanks